### PR TITLE
[ETCM-19] chainId opcode

### PR DESCRIPTION
**Pull request by mmrozek**
EIP-1344: ChainID opcode

### CODE REVIEW

**Review by KonradStaniec — COMMENTED** [review]
I would add at least one unit test maybe in `OpCodeFunSpec` ?

File: src/main/scala/io/iohk/ethereum/vm/EvmConfig.scala [hunk 1]
```diff
@@ -105,6 +108,11 @@ object EvmConfig {
     opCodeList = AghartaOpCodes
   )
 
+  val PhoenixConfigBuilder: EvmConfigBuilder = config => PhoenixConfigBuilder(config).copy(
```
Comment (KonradStaniec) [inline src/main/scala/io/iohk/ethereum/vm/EvmConfig.scala:111]: Here it shoud be:
` val PhoenixConfigBuilder: EvmConfigBuilder = config => AghartaConfigBuilder(config).copy(`

Comment (mmrozek) [reply src/main/scala/io/iohk/ethereum/vm/EvmConfig.scala:111]: Yes, my bad

**Comment by mmrozek** [comment]
@KonradStaniec, There is a generic test for all `ConstOp`s

**Comment by KonradStaniec** [comment]
ok, but then from what I see it uses byzantium config https://github.com/input-output-hk/mantis/blob/etcm-19-chainid-opcode/src/test/scala/io/iohk/ethereum/vm/OpCodeFunSpec.scala#L17, to use it we would need change this to phoenix conifg ?

**Review by KonradStaniec — APPROVED** [review]

**Pull request merged**